fresque/__init__.py

```diff
@@ -10,12 +10,27 @@
 import logging.handlers
 import os
 import sys
-
+from fresque import filters
 import flask
 import flask_fas_openid
-
+from inspect import getmembers, isfunction
 
 APP = flask.Flask(__name__)
+APP.jinja_env.trim_blocks = True
+APP.jinja_env.lstrip_blocks = True
+
+# add jinja filters
+filter_name = {
+    'shorted_commit': 'short',
+    'human_readable_time': 'humanize'
+}
+# update jinja filter with flask app env
+jinja_filters = {filter_name[name]: function
+                    for name, function in getmembers(filters)
+                    if isfunction(function)}
+
+APP.jinja_env.filters.update(jinja_filters)
```

Comment:
@pypingou that would be perfect. I was trying to do it but I ended up having circular import and so followed above way. I will look into arrow library.